packages/cli/src/index.ts

```diff
@@ -135,17 +149,15 @@ const main = async () => {
 
   // TODO - make this more intelligent
   const protocol = portal.protocols.get(ProtocolId.HistoryNetwork)
```

Comment:
An ideal solution would be to merge any ENRs provided by `args.bootnode` and `args.bootnodeList`, verify they are valid ENRs, and then loop over the protocols and add each bootnode to each protocol.  Something like:

```js
let validBootnodes = []
if (args.bootnode && isValidEnr(args.bootnode)) validBootnodes.push(args.bootnode)
if (args.bootnodeList) {
    const bootnodeData = fs.readFileSync(args.bootnodeList, 'utf-8')
    const bootnodes = bootnodeData.split('\n')
    for (const bootnode of bootnodes) {
        if (isValidEnr(bootnode)) validBootnodes.push(bootnode)
    }
}

for (const protocol of portal.protocols) {
    for (const bootnode of validBootnodes) {
    addBootNode(protocol, validBootnodes)
    }
}
```

Reply:
i've made these changes now